doc/source/whatsnew/v3.0.0.rst

```diff
@@ -409,7 +409,7 @@ Period
 
 Plotting
 ^^^^^^^^
--
+- Bug in  :meth:`.DataFrameGroupBy.boxplot` passes a ``tuple`` instead of a ``string`` when input ``DataFrame`` is pre-grouped using more than one ``column`` (:issue:`14701`)
```

Comment:
This is no longer accurate. I'd recommend something more generic, such as 

> Bug in `` :meth:`.DataFrameGroupBy.boxplot` `` failed when there were multiple groupings (:issue:`14701`)

Reply:
Correct. I will update it.